src/radical/pilot/agent/launch_method/jsrun.py

```diff
@@ -202,10 +203,11 @@ def get_launch_cmds(self, task, exec_path):
                 cmd_options += ' -r%d' % max_rs_per_node
 
             # -b: binding of tasks within a resource set (none, rs, or packed:N)
-            if td['threading_type'] == rpc.OpenMP:
-                # for OpenMP threads RP will set:
-                #    export OMP_NUM_THREADS=<cores_per_rank>
-                cmd_options += ' -b packed:%d' % cores_per_rs
+            if ranks_per_rs > 1:
+                if td['threading_type'] == rpc.OpenMP:
+                    # for OpenMP threads RP will set:
+                    #    export OMP_NUM_THREADS=<threads_per_rank>
+                    cmd_options += ' -b packed:%d' % cores_per_rank
```

Comment:
Why is the comment about `threads_per_rank` when the code uses `cores_per_rank` - which the previous version of the comment also used?

Reply:
with this I made it explicit that `cores_per_rank` here is about physical cores; and these are internal variables, not part of task description

Reply:
But the term `threads_per_rank` comes out of nowhere?  But either way, good enough for me...